packages/container/libraries/strongly-typed/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "author": "Alec Gibson",
+  "bugs": {
+    "url": "https://github.com/inversify/monorepo/issues"
+  },
+  "description": "InversifyJs strong type definitions",
+  "devDependencies": {
+    "@eslint/js": "9.13.0",
+    "@jest/globals": "29.7.0",
+    "@types/node": "20.17.4",
+    "@typescript-eslint/eslint-plugin": "8.12.2",
+    "@typescript-eslint/parser": "8.12.2",
+    "eslint": "9.13.0",
+    "jest": "29.7.0",
+    "prettier": "3.3.3",
+    "ts-jest": "29.2.5",
+    "typescript": "5.6.3"
+  },
+  "devEngines": {
+    "node": "^20.18.0",
+    "pnpm": "^9.12.1"
+  },
+  "homepage": "https://inversify.io",
+  "keywords": [
+    "dependency injection",
+    "dependency inversion",
+    "di",
+    "inversion of control container",
+    "ioc",
+    "javascript",
+    "node",
+    "typescript"
+  ],
+  "license": "MIT",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "name": "@inversifyjs/strongly-typed",
+  "os": [
+    "darwin",
+    "linux"
+  ],
```

Comment:
```suggestion
```
Let's remove this. Windows users should use this library as well (I already change this in the other packages)